M/MLX_C/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MLX_C"
+version = v"0.0.11"
+
+sources = [
+    GitSource("https://github.com/ml-explore/mlx-c.git", "126253627b95a0803273f44655fdaf5e7be4fbe1"),
```

Comment:
There is an upstream version `0.1.0`, but no upstream version `0.0.11`. I assume you're building an intermediate version? I would add a comment explaining this.

Reply:
Yeah, I had some upstream patches merged, so it was a bit early to claim that this was v0.0.11...

Ah! Just noticed that v0.1.0 was just tagged yesterday - should definitely claim that version instead.

Thanks!

Reply:
... or add a comment...

Reply:
Done (comment).

Reply:
#9898 will later on provide MLX v0.21.0, for the proper MLX C `0.1.0` version.

Reply:
Might be an idea to just wait for #9898 at this point.

Reply:
Using MLX v0.21.0 now